libqtile/widget/widgetbox.py

```diff
@@ -17,13 +17,17 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
 from __future__ import annotations
 
+from typing import TYPE_CHECKING
+
 from libqtile import bar
 from libqtile.log_utils import logger
 from libqtile.widget import Systray, base
 
+if TYPE_CHECKING:
+    from typing import Any
```

Comment:
It looks like `Any` isn't used anywhere - or I missed it?

Reply:
Will check after work but I suspect you're right.

Reply:
It's used here: https://github.com/qtile/qtile/pull/3340/files#diff-0455ac9a3773d4a03dd70d6870c17c0e213481c9794da34b01cd201d3f08be83R70

I think you still need the import even though the typing is in a `# type` line.

Reply:
Ah yes - gotcha